### DISCUSSION TITLE
触摸条型号

### DISCUSSION THREAD

**Discussion by ZhangJiaLei123** [peng-zhihui/HelloWord-Keyboard#92] [General]
请问触摸条型号是多少? 我在淘宝搜了好多, 都没有类似尺寸的。

**Comment by peng-zhihui** [comment]
自己做的啊，PCB工程里不是有源文件吗

**Reply by ZhangJiaLei123** [reply]
看到了，不好意思，没想到大佬是自己做的触摸条
![08927C44](https://user-images.githubusercontent.com/40969169/186297392-10097c06-2662-4457-add4-b421e695cee0.gif)
